### ISSUE TITLE
bug: iOS ssh apps do not work

### ISSUE THREAD

**Issue by ntimo** [warp-tech/warpgate#477]
## Description
When using iOS Terminal apps in combination with warpgate it is not possible to connect via SSH (both with password and key auth).

## Test setup
I used two apps to test this:
- Termius: https://apps.apple.com/de/app/termius-terminal-ssh-client/id549039908
- Prompt 2: https://apps.apple.com/de/app/prompt-2/id917437289

## Additional details:
Prompt does not give any error it simply stays in a connecting loop. While Termius says "END OF FILE" when trying to connect.

![8F576659-5731-4357-BC5C-4BDC4C3FB5AB](https://user-images.githubusercontent.com/6145026/201515595-7b9fb585-7041-49d1-9db6-a11d12167825.jpeg)

**Comment by Eugeny** [comment]
I've fixed it for Termius - please let me know if it works with Prompt2 too now before I spend $20 on it :P

**Comment by stappersg** [comment]
On Thu, Nov 17, 2022 at 02:35:18PM -0800, Eugene wrote:
> I've fixed it for Termius
> - please let me know if it works with Prompt2 too now

Looking forward for any
  "Yeah I care about this, so here my feedback"


> before I spend $20 on it :P

Looking forward for a
  "Donations can be transfered to bank account  IBANumber
   add 'warpgate' to the description."

(No need for setting up (and maintaining) a pay wall)



Groeten
Geert Stappers
-- 
Silence is hard to parse

**Comment by ntimo** [comment]
Hey @Eugeny I can confirm it works now with Termius thanks a lot.
